src/applications/facility-locator/components/ServiceTypeAhead.jsx

```diff
@@ -66,7 +66,7 @@ class ServiceTypeAhead extends Component {
             clearSelection();
           }
         }}
-        key={defaultSelectedItem}
+        key={defaultSelectedItem || 'defaultSelectedItem'}
```

Comment:
I'm not super familiar with this code and what `Downshift` is doing with the `key` prop, but do we need a `key` prop here at all? Usually that's only needed for array items within JSX :)

Reply:
I do not think we need it , removing it will do the fix too, should we just delete the key here?

Reply:
Is the concern that `defaultSelectedItem` can be null?

Reply:
That makes sense to me! I'd just confirm that it's not used in `Downshift` 👍Well done!

Reply:
@bkjohnson  yes